examples/kitchen-sink.yaml

```
@@ -91,6 +91,10 @@ logger_provider:
             #
             # Environment variable: OTEL_EXPORTER_OTLP_INSECURE, OTEL_EXPORTER_OTLP_LOGS_INSECURE
             insecure: false
+            # Configure retry policy
```

#nit for style symmetry. 
```suggestion
            # Configure retry policy.
```